src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/CsipRecordsController.kt

```diff
@@ -43,7 +47,7 @@ import java.util.UUID
 @Tag(name = "1. CSIP Record Controller", description = "Endpoints for CSIP Record operations")
 class CsipRecordsController(val csipRecordService: CsipRecordService, val prisonerSearchClient: PrisonerSearchClient) {
   @ResponseStatus(HttpStatus.OK)
-  @GetMapping("/prisoners/{prisonNumber}/csip-records")
+  @RequestMapping(path = ["/prisoners/csip-records"], method = [GET, POST])
   @Operation(
     summary = "Retrieve and filter all CSIP records for a prisoner.",
```

Comment:
Summary should change to "Retrieve and filter CSIP records for one or more prisoners.". The description can be removed as it doesn't add anything